src/query-builder/case-builder.ts

```diff
@@ -273,10 +293,10 @@ interface Whenable<DB, TB extends keyof DB, W, O> {
   ): CaseThenBuilder<DB, TB, W, O>
 
   /**
-   * Adds a `when` clause to the case statement, where both sides of the 
+   * Adds a `when` clause to the case statement, where both sides of the
    * operator are references to columns.
    *
```

Comment:
https://github.com/kysely-org/kysely/pull/1601/files#r2401447509

```suggestion
   *
   * See {@link when} for value-first variant.
```